test-app/config/ember-try.js

```diff
@@ -10,23 +10,19 @@ module.exports = async function () {
         name: 'ember-lts-3.28',
         npm: {
           devDependencies: {
-            'ember-source': '~3.28.0',
+            'ember-cli': '~4.12.2',
+            'ember-source': '~3.28.12',
+            '@ember/test-helpers': '^2.9.4',
+            'ember-qunit': '~6.0.0',
           },
         },
       },
       {
-        name: 'ember-lts-4.4',
+        name: 'ember-lts-4.12',
         npm: {
           devDependencies: {
-            'ember-source': '~4.4.0',
-          },
-        },
-      },
-      {
-        name: 'ember-lts-4.8',
-        npm: {
-          devDependencies: {
-            'ember-source': '~4.8.0',
+            'ember-cli': '~4.12.2',
+            'ember-source': '~4.12.0',
           },
         },
       },
```

Comment:
Have to do a little dependency shuffling to get these old ember-source versions to still pass in CI. Might be time to drop some support soon 🤷🏻